### ISSUE TITLE
[Discussion] Tellstick.conf

### ISSUE THREAD

**Issue by engelvinter** [p3tecracknell/tellprox#17]
The biggest obstacle for a beginner is still to define a proper tellstick.conf.

It would be nice if tellprox was able to autodiscover devices - and user could give them names later on - tellprox writing directly to tellstick.conf.

I have not tried it out, but the raw event callback in the telldus-core might be a way to detect devices if putting them in learning mode.

**Comment by p3tecracknell** [comment]
I don't know much about autodiscover and raw events as I don't have a telldus duo. I have been meaning to create an interface for CRUD (create/edit/update/delete) devices and groups. As I mainly use Windows, I use Telldus Center. What is a shame is that from what I can see, every piece of software that has been written for Telldus has written its own logic for dealing with creating devices the multitude of protocols and models that are supported. I will be joining them!

**Comment by engelvinter** [comment]
Ok, I have studied the Telldus Live API at Telldus and if I understand it correctly the aim of Tellprox is to provide rest-services of CRUD in the same way as Telldus Live.

So you will have to use a client like Telldus Center to access the tellprox server?

**Comment by p3tecracknell** [comment]
Yes, but there is a basic web front end which allows one to turn items on
and off and dim. I will simply extend it to allow CRUD as well.

**Comment by engelvinter** [comment]
Have you tried or received any comments on how well it works with http://remotestick.org/ nowadays?

**Comment by p3tecracknell** [comment]
Yes, I use it with Remotestick most days and for what I use it for, have no
problems. I don't have a duo/sensors so it is hard to know if that works
correctly.

**Comment by engelvinter** [comment]
By the way, I have a duo and planning to buy a temp sensor, I might try it out later during spring :-)

Is the login solution of tellprox compatible with telldus live i.e. if you enter login and password in remotestick it will work with tellprox?

**Comment by p3tecracknell** [comment]
The method by which you authenticate with Tellprox is different to that of
Telldus Live, but the author of Remotestick has kindly added support. You
simply specify the URL and the key (for authentication)
